CONTRIBUTING.md

```diff
@@ -20,7 +23,7 @@ The development branch is <code>main</code>. All pull request should be made aga
 - Create a new branch (include the issue id and something readable):
 
   ```sh
-  git checkout -b doc-999-my-feature-or-fix
+  git checkout -b doc-999-feat-somefeature-that-rocks
```

Comment:
I prefer `feat/doc-999-some-feature-that-rocks`

Reply:
fine for me

Reply:
@Mythie merged main. we stop using the slack slug for updating? either way ready to merge

Reply:
We're definitely using `documen.so/slack` not sure why you've gotten an old one here. Please update as part of this 😄

Reply:
zoink

Reply:
boing